src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PremisesEntity.kt

```diff
@@ -20,9 +20,12 @@ data class PremisesEntity(
   val id: UUID,
   val name: String,
   val apCode: String,
+  val address: String,
   var postcode: String,
   var totalBeds: Int,
   val deliusTeamCode: String,
+  val service: String,
+  val notes: String,
```

Comment:
Is there a data type such as Text that better supports longer form data than String? 

Edit: I see later the migration adds a field with `TEXT` so this might be ok.